modules/wri_event/wri_event.install

```diff
@@ -193,3 +193,29 @@ function wri_event_update_10004() {
     ]);
   }
 }
+
+/**
+ * Add a new calendar event description field to events.
+ */
+function wri_event_update_10005() {
+  // Add the fields.
+  \Drupal::service('distro_helper.updates')
+    ->installConfig('field.storage.node.field_calendar_description', 'wri_event', 'install', 'TRUE');
+
+  // Attach the fields to the Event node.
+  \Drupal::service('distro_helper.updates')
+    ->installConfig('field.field.node.event.field_calendar_description', 'wri_event', 'install', 'TRUE');
```

Comment:
Since we're still using the old-fashioned pantheon commands `drush updb` then `drush cim` this new field is getting deleted and re-created post-update. Can you update Distro helper to call the `syncUUIDs` command right before the config gets auto-exported [here](https://github.com/thinkshout/distro_helper/blob/main/src/DistroHelperUpdates.php#L151)? Then open a new branch and we'll test it out on this repo.